modules/ROOT/pages/endpoint-sftp-receive-target.adoc

```diff
@@ -1,84 +1,88 @@
 = SFTP Receive Endpoint Settings
 
-SFTP receive endpoint settings determine where to receive messages in the following contexts:
+SFTP receive endpoints configure where to receive B2B messages transmitted over SFTP. There are two types of SFTP receive endpoints:
 
-* Receive from partners
+* Receive from Partners
 +
-Source endpoint in an inbound message flow that receives B2B EDI messages from partners either directly or via third-party connections
+Source endpoint in an inbound message flow that receives B2B messages from partners either directly or via third-party connections.
 +
-* Source at host
+A host, partner, or third-party connection can own SFTP Receive from Partners endpoints.
 +
-Source endpoint in an outbound message flow that receives JSON and XML messages from the backend applications
-
-A host, partner, or third-party connection can own SFTP receive endpoints.
+* Source at Host
++
+Source endpoint in an outbound message flow that receives messages from the backend applications.
++
+Only a host can own SFTP Source at Host endpoints.
 
-Create SFTP receive endpoints by configuring these settings on the host or Partner page:
+Create SFTP receive endpoints by configuring these settings on the *<host>* or *Partner* page:
 
 [%header%autowidth.spread]
 |===
 |Name |Description | Required
 | Endpoint Name
 | Automatically generated name used to identify the endpoint, which you can subsequently modify manually.
-| Yes
+| x
 
 | Owner
 | Partner or host that owns the endpoint.
-| Yes
+| x
 
 | Protocol
-| Protocol selected in the *Protocol* field of the *New endpoint*. The value in this field cannot be modified.
-| Yes
+| Protocol that the endpoint supports.
+| x
 
 | Description
 | User-supplied value that describes the purpose of the endpoint.
-| No
+|
 
 | Host
 | SFTP server host, such as `sftp.mulesoft.com`, `localhost`, or `192.168.0.1`. The host must be accessible from your Mule environment.
-| Yes
+| x
 
 | Port
 | Port number of the SFTP server.
-| Yes
+| x
 
 | Username
 | Username for the SFTP server.
-| Yes, if the server is authenticated.
+| x (if the server is authenticated)
 
 | Password
 | Password for the SFTP server.
-| Yes, if the server is authenticated.
+| x (if the server is authenticated)
 
 | Source path
-| Path on the SFTP server to which your trading partner sends the inbound files.
-| Yes
+| Path on the SFTP server to which your partner sends the inbound files.
+| x
 
 | Polling Frequency
-| Frequency in which the server checks the source path for new files. The value in this field is `1 minute` by default.
-| Yes
+| Frequency in which the server checks the source path for new files. The default value is `1 minute`.
```

Comment:
THIS is a very interesting preposition use issue, on which I want to do more research. However, I believe that you want to use "at" (my choice) or "with" rather than "in."
Here's a snippet of what I found, but several people commented to the effect that preposition choice is often based on custom rather than rules:

"'Frequency' is a noun, and you want to modify the verb phrase 'makes purchases,' so you want an adverbial phrase or clause. We can provide this with 'at which' or 'with which':

...they can decrease the frequency with which a customer makes purchases from the site.

We could also replace 'frequency' with the related adverb 'frequently'

...they can decrease how frequently a customer makes purchases from the site.

There are other variations, but these are the two that make the least change to the original."